.github/aur/flux-bin/.SRCINFO.template

```diff
@@ -0,0 +1,20 @@
+pkgbase = flux-bin
+	pkgdesc = Open and extensible continuous delivery solution for Kubernetes
+	pkgver = ${PKGVER}
+	pkgrel = ${PKGREL}
+	url = https://fluxcd.io/
+	arch = x86_64
+	arch = armv6h
+	arch = armv7h
+	arch = aarch64
+	license = APACHE
+	optdepends = kubectl
+	provides = flux-cli
+	conflicts = flux-git
+	conflicts = flux-go
```

Comment:
I think these need an update too now that the `PKGBUILD.template` files have changed.

Reply:
:eagle: :eyes:

Ugh, you see my point about wanting to have `makepkg` generate these? :smile: